docs/rules/prefer-reflect.md

```diff
@@ -54,12 +54,12 @@ Examples of **correct** code for this rule when used without exceptions:
 ```js
 /*eslint prefer-reflect: "error"*/
 
-Reflect.apply(undefined, args);
-Reflect.apply(null, args);
+Reflect.apply(obj.foo, undefined, args);
```

Comment:
It might be worthwhile rewriting the above incorrect patterns to use a mixture of names, rather than `foo` for everything.